### PR TITLE
Added aliasing

### PR DESCRIPTION
**Pull request by kkhsing**
#### Short description of what this resolves:
These changes add aliasing features. From hand testing, seems to handle requests properly except for rlist and ideas which I couldn't seem to get to work even before I made any changes.
#### Changes proposed in this pull request:
alias module along with a couple helper functions in util. Changes the way arguments are passed so I'm not sure if this is the best solution.

Usage
$ yoda alias new "flashcards select" "fs"
$ yoda fs english
and
$ yoda alias delete fs

**Fixes**: #

### CODE REVIEW

File: modules/alias.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import click
+import dev
```
Comment (manparvesh) [inline modules/alias.py]: This line seems to be causing [build failure for Python 3](https://travis-ci.org/yoda-pa/yoda/jobs/365879074#L1412). It seems unused to me, so I think just removing this should fix it.

**Comment by manparvesh** [comment]
Hi @kkhsing 
This is great work!
However, it would be very helpful if you could fix the build issue, and also if you could add instructions to use this command in the README.
Thanks a lot for contributing! :smile:

File: modules/alias.py (hunk 1 above)
Comment (kkhsing) [reply modules/alias.py]: I've made the changes along with a couple others. I'm still getting a build failure for Python 3, do you mind helping me understand where the error is coming from?

**Comment by manparvesh** [comment]
@kkhsing 
The MR looks great and would be a great addition to the project, so I am merging your code. I've created an issue (#104) for fixing the build issues, so if you get time to contribute again, you can take a look at #104 or some other issue that you would like to resolve.
Once again, thanks for contributing! :smile:

**Pull request merged**